github_conf/branch_protection_rules.json

```diff
@@ -0,0 +1,4 @@
+{
+    "message": "Not Found",
+    "documentation_url": "https://docs.github.com/rest"
+}
```

Comment:
🤔 what is this for?

Reply:
I didn't make this update. Perhaps it got merged into the branch during a pull.